main.c

```diff
@@ -31,7 +31,9 @@
 #include <shlwapi.h>
 #include <wtsapi32.h>
 #include <prsht.h>
+#ifndef _MSC_VER
 #include <pbt.h>
```

Comment:
pbt.h is an obsolete header  -- I think we can just remove it. May be as a separate commit?

Reply:
moved into separate commit